### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by vanhauser-thc**
small addition to help people to use afl.rs more effectively.

also note I just released afl++ 4.08c :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ You can use the new command now, if you like. If the binary is already installed
 
 Documentation can be found in the [Rust Fuzz Book](https://rust-fuzz.github.io/book/afl.html).
 
+## Hints
+
+By default the AFL++ CMPLOG feature is activated, which is an amazing feature to achieve good code coverage.
```
Comment (smoelius) [inline README.md:27]:
```suggestion
By default, the AFL++ [CMPLOG](https://github.com/AFLplusplus/AFLplusplus/blob/stable/instrumentation/README.cmplog.md) feature is activated, which helps to achieve good code coverage.
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ You can use the new command now, if you like. If the binary is already installed
...
+However it is not beneficial to activate CMPLOG on more than 2 instances.
```
Comment (smoelius) [inline README.md:28]:
```suggestion
However, it is not beneficial to activate CMPLOG on more than two instances.
```

File: README.md [hunk 3]
```diff
@@ -22,6 +22,14 @@ You can use the new command now, if you like. If the binary is already installed
...
+So if you run multiple AFL++ instances on your fuzzing target, you can disable CMPLOG by specifying the command line parameter '-c -'.
+
+It is highly recommended to familiarize yourself with the AFL++ features and how to successfully run a fuzzing campaign. This [document](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/fuzzing_in_depth.md) will help you getting started.
```
Comment (smoelius) [inline README.md:31]:
```suggestion
This [document](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/fuzzing_in_depth.md) will familiarize you with AFL++ features to help in running a successful fuzzing campaign.
```

**Review by smoelius — CHANGES_REQUESTED** [review]
Thank you very much for this.

All of my suggestions are very nitpicky, and mostly about reducing the number words.

Though, may I please ask that this section be positioned between the "What does it look like?" and "`lazy_static` variables" sections? (I am trying be respectful of the previous maintainers' content.)

I will publish a new version with these changes and afl++ 4.08c within a day or so.

**Comment by vanhauser-thc** [comment]
sure, no problem!
is it fine now?

**Review by smoelius — APPROVED** [review]

**Comment by smoelius** [comment]
Yes! Thanks tremendously!

**Pull request merged**